WinUIGallery/ItemPage.xaml.cs

```diff
@@ -187,7 +187,7 @@ protected override async void OnNavigatedTo(NavigationEventArgs e)
                     this.contentFrame.Navigate(pageType);
                 }
 
-                args.NavigationRootPage.NavigationView.Header = item?.Title;
+                args.NavigationRootPage.NavigationView.Header = item;
```

Comment:
Not clear why this is needed.

Reply:
The binding of properties in the headertemplate previously bound to the datacontext itsself instead of a property. Now, we need multiple properties of the item and as such, the datacontext for the binding needs to be the item itsself.